### PR TITLE
Professor Dropdown

### PR DESCRIPTION
**Pull request by DC00**
Functionality is fine. Styling is a B+

### CODE REVIEW

**Comment by DC00** [comment]
There's a problem with the dropdown on Safari

**Comment by DC00** [comment]
And Firefox...

**Comment by aw3as** [comment]
should be finished please look over

**Comment by DC00** [comment]
deptshowerror

**Comment by DC00** [comment]
Do I need to update my database

**Comment by DC00** [comment]
Looks fine

File: app/assets/stylesheets/courses.scss [hunk 1]
```diff
@@ -496,6 +496,18 @@ p {
...
+.professors-switcher {
+  font-size: 24px;
+  border: 0px;
+  outline: 0px;
+  width: auto;
+  background-color: #EEE;
+  border-color: #EEE;
+  box-shadow: inset 0 0px 0px rgba(0, 0, 0, 0.075);
+  font-weight: 300;
+  margin-left: -7px;
```
Comment (msukkar) [inline app/assets/stylesheets/courses.scss]: negative margin?

**Comment by msukkar** [comment]
bottom half of text is cut off in chrome
image

Also, how it looks on firefox: 
image

**Pull request merged**